Update Object_ProBuilderize.md (#569)
Unity-Technologies/com.unity.probuilder: Documentation~/Object_ProBuilderize.md

# ProBuilderize a standard GameObject

> You can turn any 3D GameObject to a ProBuilder object. 

Author: 3D object is a specific category in the Create New menu, so I assumed it was well known. Terrain isn't under that list. However, we can change it to "any 3D GameObject that has the Mesh Filter and Mesh Renderer components".

> You can turn any 3D GameObject to a ProBuilder object. 

Author: ```suggestion
You can turn any 3D GameObject that has the Mesh Filter and Mesh Renderer components to a ProBuilder object. 
```

> You can turn any 3D GameObject to a ProBuilder object. 

Reviewer: suggestion looks good 👍

> You can turn any 3D GameObject to a ProBuilder object. 

Reviewer: ```suggestion
You can turn a 3D GameObject that has the Mesh Filter and Mesh Renderer components to a ProBuilder object.
```
I'd go for this as it no longer says "any" (I'd rather not overpromise).

> You can turn any 3D GameObject to a ProBuilder object. 

Author: Remove "any" and committed

To ProBuilderize a GameObject:

1. In the **Scene** view, select the GameObject.
2. Do one of the following:
    * From the main menu, select **Tools** > **ProBuilder** > **Object** > **Pro Builderize**.
    * Right-click the GameObject and select **MeshFilter** > **ProBuilderize**.
3. The **ProBuilderize** overlay opens, and you can customize the ProBuilder object.

## ProBuilderize options

Use the options in the **ProBuilderize** overlay to customize the ProBuilder object.

| **Property** | **Description** |
| :--- | :--- |

> | **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles where possible.<!--can we give an example of where it won't be possible?--> |

Author: You didn't misunderstand the question :)
Out of curiosity, why would people choose quads or triangles? Why do we provide both options?

> | **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles where possible.<!--can we give an example of where it won't be possible?--> |

Author: ```suggestion
| **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles. |
```

> | **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles where possible.<!--can we give an example of where it won't be possible?--> |

Author: Please see the updated suggestion above.

> | **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles where possible.<!--can we give an example of where it won't be possible?--> |

Reviewer: @suearkinunity Hmm, I would maybe not single out `animation` and just explain it more broadly. Something along the lines of:
`While quads are generally more practical to manipulate, the build process can result in topology that might not fully match the triangle topology of the original asset - this can lead to unexpected visual artifacts, deformation issues and so on. To build the faces of triangles, disable **Import Quads**.` .

| **Import Smoothing** | Smooth the transition between faces to make a softer object. For more information, refer to [Smoothing groups](smoothing-groups.md). |
| **Smoothing Threshold** | Decide which adjacent faces to add to each smoothing group. Use a value larger than the angle between the faces you want to add. The range is from 0.0001 to 45. This setting is only available when you use **Import Smoothing**. |

> <!--Right click > MeshFilter > ProBuilderize - the overlay doesn't have the Live Preview option. This sounds wrong.-->

Author: ```suggestion
```

> <!--Right click > MeshFilter > ProBuilderize - the overlay doesn't have the Live Preview option. This sounds wrong.-->

Author: I meant the checkbox that lets you turn live preview on and off. It's not there. Should it be?

> <!--Right click > MeshFilter > ProBuilderize - the overlay doesn't have the Live Preview option. This sounds wrong.-->

Reviewer: Oh sorry, yeah I don't think it should have it as it can be a pretty expensive operation so live-previewing it would be tough.
